sklearn/metrics/cluster/_supervised.py

```diff
@@ -740,10 +740,10 @@ def mutual_info_score(labels_true, labels_pred, *, contingency=None):
     Parameters
     ----------
     labels_true : int array, shape = [n_samples]
-        A clustering of the data into disjoint subsets.
+        A clustering of the data into disjoint subsets (U).
```

Comment:
I think it could be nice to have something like

```suggestion
        A clustering of the data into disjoint subsets, called $U$ in the above formula.
```